website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md

```diff
@@ -0,0 +1,649 @@
+|3. **dbt Semantic Layer as a data source**  | Connect to the dbt Semantic Layer as a data source (for example, the Snowflake Proxy Server). Users can execute dbt-SQL to query metrics or models and use macros.* | ✅ |
+|4. **Query metrics**  | Query the imported metrics via a metric-centric UI (for example, a user can select a metric, time grain, and dimensions of interest). | ✅ |
+|5. **Entity definitions**   | Import/sync entity definitions (descriptions, dimensions, data types, relationships, metrics, and more) and query entities via the dbt Semantic Layer. | _*Coming soon, see the [Product Roadmap](#product-roadmap) for details_ |
+|6. **dbt Semantic Layer Connector**   | A dedicated connector with the ability to query any data platform supported in dbt Cloud. (Will replace (3).) | _*Coming soon, see the [Product Roadmap](#product-roadmap) for details_ |
+
+_*The coming soon features above are expected to launch in 2023. Review the [Product Roadmap](#product-roadmap) for more details and timelines._
+
+### Integration best practice
+
+To build a successful and seamless dbt Semantic Layer integration, it should express the following:
+
+- **Consistent**: Have a consistent user experience (UX) incorporated into existing core user workflows.
+- **Trustworthy**: Treat dbt assets (metrics, models, and entities) as first-class objects and indicate that their definitions and resulting datasets come from dbt Cloud.
+- **Efficient**: Provide a clear advantage over the current approach to setting up metrics and analyses, and finding dimensions/datasets in the tool.
+- **Accessible**: Include a self-serve component so a data consumer can ask questions via the user interface (UI), if applicable.
+
+
+## Use the Metadata API
+
+To get started, you must meet the dbt Semantic Layer [prerequisites](/docs/use-dbt-semantic-layer/dbt-semantic-layer#prerequisites). This section will explain how to connect to and query the [Metadata API](/docs/dbt-cloud-apis/metadata-api) for model and metric definitions.
+
+**Metadata API authorization**
+
+To learn how to authorize requests to the Metadata API, review the [documentation](/docs/dbt-cloud-apis/metadata-querying#authorization) for more details. Metrics-specific queries work identically to existing Metadata API queries, so existing integrations used to fetch model metadata will work perfectly in the context of metrics. 
+
+**Query the Metadata API**
+
+Test out the Metadata API by using the [GraphQL sandbox](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fmetadata.cloud.getdbt.com%2Fgraphql) and use this [Python client](https://github.com/transform-data/dbt-metadata-client) as a starting point to develop. 
+
+
+### Fetch models for a project
+
+You can fetch and query models, or details about a specific model, for a project from a given job.
+
+<!--- tabs for listing models and fetching details about specific model --->
+<Tabs>
+
+<TabItem value="list" label="Listing models">
```

Comment:
Split this tab's contents:
- Listing models for a job
- Example response with models